### PR TITLE
Remove a few unneeded header inclusions in reco packages

### PR DESCRIPTION
**Pull request by perrotta**
#### PR description:

Some sparse cleaning of a few header inclusions

#### PR validation:

It builds

### CODE REVIEW

File: RecoHI/HiJetAlgos/plugins/HiFJRhoFlowModulationProducer.cc [hunk 1]
```diff
@@ -72,8 +71,8 @@ HiFJRhoFlowModulationProducer::HiFJRhoFlowModulationProducer(const edm::Paramete
       evtPlaneToken_(consumes<reco::EvtPlaneCollection>(iConfig.getParameter<edm::InputTag>("EvtPlane"))) {
   produces<std::vector<double>>("rhoFlowFitParams");
   TMinuitMinimizer::UseStaticMinuit(false);
-  lineFit_p_ = std::unique_ptr<TF1>(new TF1("lineFit", lineFunction, -TMath::Pi(), TMath::Pi()));
-  flowFit_p_ = std::unique_ptr<TF1>(new TF1("flowFit", flowFunction, -TMath::Pi(), TMath::Pi()));
+  lineFit_p_ = std::make_unique<TF1>("lineFit", lineFunction, -TMath::Pi(), TMath::Pi());
```
Comment (perrotta) [inline RecoHI/HiJetAlgos/plugins/HiFJRhoFlowModulationProducer.cc:74]: This update was automatically implemented by "scram b code-checks"

**Comment by perrotta** [comment]
please test

**Comment by perrotta** [comment]
+1
- Technical, cleaning
- Jenkins tests pass and show no effect on outputs

**Comment by qliphy** [comment]
+1

**Pull request merged**